build.gradle

```diff
@@ -137,7 +136,11 @@ subprojects {
     plugin 'org.gradle.test-retry'
   }
 
-  sourceCompatibility = 1.8
+  if (JavaVersion.current() >= JavaVersion.VERSION_1_9) {
+    tasks.withType(JavaCompile).configureEach {
+      it.options.release = 8
+    }
```

Comment:
This can be just written as:
  tasks.withType(JavaCompile) {
    options.release = 8
  }

Reply:
Thanks for the tip